src/custom/components/swap/FeeInformationTooltip.tsx

```diff
@@ -119,7 +122,7 @@ export default function FeeInformationTooltip(props: FeeInformationTooltipProps)
           }
         />
       </span>
-      <FeeAmountAndFiat>
+      <FeeAmountAndFiat title={`${fullFeeAmount || '-'} ${symbol || ''}`}>
```

Comment:
i believe you can set defaults to the initial const declaration for both of these inside the destructured array return on L 81 but I may be wrong